daemon/inertiad/project/data.go

```diff
@@ -138,6 +140,49 @@ func (c *DeploymentDataManager) GetEnvVariables(decrypt bool) ([]string, error)
 	return envs, err
 }
 
+// AddProjectBuildData stores and tracks metadata from successful builds
+// TODO: definitely need to add more context to errors
+func (c *DeploymentDataManager) AddProjectBuildData(projectName string, mdata DeploymentMetadata) error {
```

Comment:
@bobheadxi  you mind taking a look at the way I'm updating the db? Still getting a deadlock when adding two projects with the same name

Reply:
Nvm 😂